### DISCUSSION TITLE
Grouping or Gangs

### DISCUSSION THREAD

**Discussion by plusflabby** [EveronLife/EveronLife#63] [Ideas]
We've had gangs in arma life for a bit and I propose something like what follows:  A system defined by a config for each faction to decide whether players may make their own group, A group that may automatically assign a player from a specific faction, and player tags (name above head) for group members.
Thoughts, concerns, or rebuttals?

**Comment by Arkensor** [comment]
For the endgame content, we have some kind of gang/group system in mind. But I am not sure why those should be automatic. 
Or do you mean if you join a group you get a tag prefix automatically?

I thought people create a group, and invite others. And then the group can own shared houses and have other coop benefits. A gang would be a group that is known for doing criminal things, but I guess they might be the same from a technical standpoint?

**Reply by plusflabby** [reply]
I was thinking let's say the police faction is one big group and new police get added to it. For group members you get a name tag where your character is, like default. Also you may have them show up on map.

**Comment by plusflabby** [comment]
> I thought people create a group, and invite others. And then the group can own shared houses and have other coop benefits. A gang would be a group that is known for doing criminal things, but I guess they might be the same from a technical standpoint?

Shared houses would be great and maybe shared garages too.
